src/module/trainee/types.graphql

```diff
@@ -0,0 +1,42 @@
+type data {
+    count: Int
+    records: [user]!
+}
+
+type AllTrainee {
+   message: String
+   status: String
+   data: data
```

Comment:
Please use meaningful names for schema.
Go through the schema and rename these wherever necessary